export const getProduct = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
